win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -1015,7 +844,6 @@ static DWORD WINAPI chaos_thread_with_timers_no_lock_func(LPVOID lpThreadParamet
                     (void)InterlockedIncrement64(&chaos_test_data->expected_call_count);
                 }
             }
-            break;
```

Comment:
>break; [](http://example.com/codeflow?start=12&length=6)

We should leave this break; in, generally each case: should have its own, break. No need to remove it.

Reply:
Fixed.